Package@swift-5.6.swift

```diff
@@ -28,7 +28,9 @@ var package = Package(
         .target(
             name: "ArgumentParser",
             dependencies: ["ArgumentParserToolInfo"],
-            exclude: ["CMakeLists.txt"]),
+            exclude: ["CMakeLists.txt"],
+            swiftSettings: [.unsafeFlags(["-Xfrontend", "-strict-concurrency=complete"])]
```

Comment:
This is a "very bad idea" ™️ - anyone who pulls in the 5.6 manifest will see this propagated to their library or application and potentially introduce compiler errors and warnings.

We solved this in Vapor with a 5.9 specific manifest using the experimental feature flag which restricts it to that package - e.g. https://github.com/vapor/vapor/blob/main/Package%40swift-5.9.swift#L96

Reply:
Definitely! Thanks for that note — I was planning to just remove the flag, and didn't realize that `enableExperimentalFeature` allows us to keep the setting package-local.